### PR TITLE
Ravinder | giddh 2.0 | Fixed bugs

### PR DESCRIPTION
**Pull request by ravinderwalkover**
Ravinder | giddh 2.0 | Fixed bugs

### CODE REVIEW

File: apps/web-giddh/src/app/invoice/preview/invoice.preview.component.scss [hunk 1]
```diff
@@ -351,6 +351,10 @@ ul.selectActionDropdown {
     display: none;
 }
 
+.advance-icon {
```
Comment (rohit-walkover) [inline apps/web-giddh/src/app/invoice/preview/invoice.preview.component.scss:354]: redundant css code use **cursor-pointer** class

File: cypress/config/production.json [hunk 2]
```diff
@@ -0,0 +1,11 @@
+{
+  "baseUrl": "https://giddh.com/login",
+  "pageLoadTimeout" : 80000,
+  "env": {
+    "url": "https://giddh.com/login",
+    "dashBoardUrl" : "https://app.giddh.com/pages/home",
+    "authKey" : "vMvGKUiRYvyZqvwhljopR3R6-5XqBd-JWYu7GmO3xiZp-ySdHroKfx5cxM1KVsrOyz2HaL3DWYL6ko8yofhfsyHdp3a9MHQCJNlWFtdDLe0=",
```
Comment (prateek-giddh) [inline cypress/config/production.json:7]: Please check if it is safe to put auth key in code

**Review by prateek-giddh — CHANGES_REQUESTED** [review]

File: package.json [hunk 3]
```diff
@@ -55,20 +55,31 @@
     "package": "npm run build.electron.giddh && cd dist/apps/web-giddh-prod && npx  electron-builder build --config ./../../../electron-sign/electron-builder.json  -mw --x64",
     "package:linux": "npm run build.electron.giddh && cd dist/apps/web-giddh-prod && npx electron-builder  build --config ./../../../electron-sign/electron-builder.json  --linux --x64",
     "package:mac": "npm run build.electron.giddh && cd dist/apps/web-giddh-prod && npx  electron-builder build --config ./../../../electron-sign/electron-builder.json  --macos --x64",
-    "package:windows": "npm run build.electron.giddh && cd dist/apps/web-giddh-prod && cross-env AWS_ACCESS_KEY_ID=key_id AWS_SECRET_ACCESS_KEY=access_key npx electron-builder build  --config ./../../../electron-sign/electron-builder.json --win --ia32 --x64 "
+    "package:windows": "npm run build.electron.giddh && cd dist/apps/web-giddh-prod && cross-env AWS_ACCESS_KEY_ID=key_id AWS_SECRET_ACCESS_KEY=access_key npx electron-builder build  --config ./../../../electron-sign/electron-builder.json --win --ia32 --x64 ",
```
Comment (rohit-walkover) [inline package.json:58]: can't we use other mapping key value for cypress build

Comment (ravinderwalkover) [reply package.json:58]: why?

File: cypress/config/production.json (hunk 2 above)
Comment (ravinderwalkover) [reply cypress/config/production.json:7]: This is Giddh Auth key, so there is no issue.

**Review by rohit-walkover — APPROVED** [review]

**Pull request merged**